plugin/accel_embree/embree.cpp

```diff
@@ -0,0 +1,66 @@
+#include "embree.h"
+#include <lm/json.h>
+LM_NAMESPACE_BEGIN(LM_NAMESPACE)
+
+#define tryAt(j,str,v) try{j.at(str).get_to(v);}catch(...){ v=v;}

```

Comment:
tryAt macro can be replaced with lm::json::value functions, which can accept a default value.